### PR TITLE
Add benchmark for loading chunks and chunk descs.

### PR DESCRIPTION
**Pull request by beorn7**
@juliusv 

This is in preparation for testing out some ideas.

### CODE REVIEW

File: storage/local/persistence_test.go [hunk 1]
```diff
@@ -884,3 +884,74 @@ func verifyIndexedState(i int, t *testing.T, b incrementalBatch, indexedFpsToMet
...
+	"b004b821ca50ba26",
+	"b037c21e884e4fc5",
+	"b037de1e884e5469",
+}
+
+func BenchmarkLoadChunksSequentially(b *testing.B) {
+	p := persistence{
+		basePath: "fixtures",
+	}
+	sequentialIndexes := make([]int, 47)
```
Comment (juliusv) [inline storage/local/persistence_test.go]: May I ask: why exactly 47?

Comment (juliusv) [inline storage/local/persistence_test.go]: Ah, because that's in the fixture, I guess...

File: storage/local/persistence_test.go [hunk 2]
```diff
@@ -884,3 +884,74 @@ func verifyIndexedState(i int, t *testing.T, b incrementalBatch, indexedFpsToMet
...
+			}
+		}
+	}
+}
+
+func BenchmarkLoadChunksRandomly(b *testing.B) {
+	p := persistence{
+		basePath: "fixtures",
+	}
+	randomIndexes := []int{1, 5, 6, 8, 11, 14, 18, 23, 29, 33, 42, 46}
```
Comment (juliusv) [inline storage/local/persistence_test.go]: Would one not want to load all 47 chunks here too (randomly), unless you don't care about comparing it to the benchmark above?

Comment (beorn7) [inline storage/local/persistence_test.go]: Random order is not a realistic scenario. AFAIK, the load is either completely sequential as above, or it has "gaps", which I try to simulate here.

No comparison of the two benchmarks intended. This is all only to play with the load methods later.

**Comment by juliusv** [comment]
Ok then. :+1:

**Pull request merged**